pkg/combustion/kubernetes.go

```diff
@@ -316,11 +317,18 @@ func (c *Combustion) downloadRKE2Artefacts(ctx *image.Context, cluster *kubernet
 }
 
 func kubernetesVIPManifest(k *image.Kubernetes) (string, error) {
+	ip, err := netip.ParseAddr(k.Network.APIVIP)
+	if err != nil {
+		return "", fmt.Errorf("parsing kubernetes APIVIP address: %w", err)
```

Comment:
I feel like this should read as `apiVIP` instead of all caps, since that's what the field is named in the definition. But I don't feel particularly strongly if you decide to keep it like this.